module_11/index.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+require "data/users.php";
+require "data/passwords.php";
+
+
+?>
+<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <link href="styles.css" rel="stylesheet">
+    <title>Project - ведение списков</title>
+    <link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
+</head>
+
+<body>
+
+<div class="header">
+    <div class="logo"><img src="i/logo.png" alt="Project"></div>
+    <div class="author">Автор: <span class="author__name">***</span></div>
+</div>
+
+<div class="clear">
+    <ul class="main-menu">
+        <li><a href="#">Главная</a></li>
+        <li><a href="#">О нас</a></li>
+        <li><a href="#">Контакты</a></li>
+        <li><a href="#">Новости</a></li>
+        <li><a href="#">Каталог</a></li>
+    </ul>
+</div>
+
+<table width="100%" border="0" cellspacing="0" cellpadding="0">
+    <tr>
+        <td class="left-collum-index">
+
+            <h1>Возможности проекта —</h1>
+            <p>Вести свои личные списки, например покупки в магазине, цели,
+                задачи и многое другое. Делится списками с друзьями и
+                просматривать списки друзей.</p>
+
+
+        </td>
+        <td class="right-collum-index">
```

Comment:
Hide that item if authrization passed successfully

Reply:
Here you can also check `authorised` variable and hide the item using `hidden` option for tag `<td class="right-collum-index">`